lab-doug/gulpfile.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const nodemon = require('nodemon');
+const mocha = require('gulp-mocha');
+
+const paths = ['*.js', 'lib/*.js', 'model/*.js', 'route/*.js', 'test/*.js'];
+
+
+gulp.task('eslint', function(){
+  gulp.src(paths)
+  .pipe(eslint())
+  .pipe(eslint.format())
+  .pipe(eslint.failAfterError());
```

Comment:
This is the line that is breaking your tests when run through gulp. For debugging you can get rid of it/comment it out, but in reality you should just fix your linting errors.